### ISSUE TITLE
Nothing happens on windows7

### ISSUE THREAD

**Issue by Dequisitor** [extr0py/oni#54]
Hi,

After `npm install -g oni` I used `oni` -> nothing happens, after a second or so the focus is back on the terminal (the cursor blinks - don't know the technical term).
So I did a `npm install -g oni-vim` and `oni-vim` -> same result.

**oni** -> 0.0.1
**oni-vim** -> 0.1.2
**npm** -> v 4.0.3 (just upgraded from 3.10.2)
**electron** -> 1.4.11 (only under oni-vim)

Thanks in advance,
dequisitor

P.S.: thanks for this awesome GUI

**Comment by extr0py** [comment]
Thanks for the report, @Dequisitor !

A couple follow-up questions:
1) What node version are you running? I'm assuming its 7+ based on the npm version, but wanted to check
2) Are you on 32-bit or 64-bit Windows?

It sounds like there are some related reports:
https://github.com/electron/electron/issues/2126

My hunch is it might be worth trying the 32-bit electron bits and see how they work

**Comment by Dequisitor** [comment]
Hi, @extr0py 

Thanks for the quick answer. 
1. My node version was 4.4.5 when I submitted the bug report. Updated it to 6.9.2, still the same situation.
2. 64-bit windows.

After reading bug report you referenced (thanks for that too) I realized I should have mentioned this is a workstation at my company, which means there can be a couple of programs keeping oni from running. Although electron starts without issues. Will try 32bit electron tomorrow.

Never tried electron, so this might be stupid: cd-d into oni directory, issued `electron .`

```
$ electron .

App threw an error during load
ReferenceError: document is not defined
    at C:\blabla\AppData\Roaming\npm\node_modules\oni\node_modules\screenfull\dist\screenfull.js:69:25
    at C:\blabla\AppData\Roaming\npm\node_modules\oni\node_modules\screenfull\dist\screenfull.js:78:4
    at Object.<anonymous> (C:\blabla\AppData\Roaming\npm\node_modules\oni\node_modules\screenfull\dist\screenfull.js:155:3)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)

Thanks for you help.

Have a nice day,
dequisitor

```

**Comment by extr0py** [comment]
Hi @Dequisitor  - there was a bug with this package where the built files weren't included... so that is likely the culprit. I've published a new package v0.1.3 - try that out and see if it works better!

**Comment by Rafi993** [comment]
I am using windows 10  X64 now nothing happens when  I type oni after install from cmd. Hope it gets fixed

**Comment by Rafi993** [comment]
I tried both but the same result.

**Comment by Rafi993** [comment]
When I updated the electron version to 1.4.12 the problem was solved

**Comment by extr0py** [comment]
@Rafi993 - thanks for the info! Glad it's working now.

**Comment by Dequisitor** [comment]
Hi @extr0py - sorry for the late response, I was out for the weekend. 

Thanks for sorting out the issue with @Rafi993. Sadly, after an update (just sheer `npm install -g oni oni-vim`) oni and oni-vim commands still seem to do nothing. So I checked the versions, installed electron manually, and tried `electron .` in the oni and oni-vim directories: `oni/electron .` resulted in the previously copy/pasted error, and `oni-vim/electron .` resulted in 'not an electron app' error. (again I'd like to mention I know almost nothing about electron) 

**Solution:** So I checked the main.js which is definitely an electron file, I issued `electron main.js` and that worked just fine. 

99% sure my setup is the one to blame for not being able to run oni and oni-vim, but I thought I'd mention this just in case anyone has the same problem.
